Sources/SourceKit/sourcekitd/SwiftLanguageServer.swift

```diff
@@ -383,20 +390,18 @@ extension SwiftLanguageServer {
     _ = handle
   }
 
-  func rewriteCompletionPlacholders(_ completion: String) -> String {
-    if !completion.contains("<#") {
-      return completion
-    }
-
-    var result = completion
+  func rewriteSourceKitPlaceholders(_ skPlaceholder: String, clientSupportsSnippets: Bool) -> String {
```

Comment:
The reasoning to make this method generic is because they'll likely add snippet support to WorkspaceEdits in the near future. The refactors can use it as well when it happens.